tests/test_failure_propagator.py

```diff
@@ -1,15 +1,59 @@
 import pytest
 
-from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import (
+    acceptance,
+    tier1,
+    tier2,
+    tier3,
+    tier4a,
+    tier4b,
+    tier4c,
+)
 
 
 @tier1
-@pytest.mark.last
+@acceptance
+@tier2
+@tier3
+@tier4a
+@tier4b
+@tier4c
 class TestFailurePropagator:
     """
     Test class for failure propagator test case. The test intention is to run last and propagate
     teardown failures caught during the test execution, so regular test cases won't false negatively fail
     """
 
+    @pytest.mark.second_to_last
+    def test_report_skip_triggering_test(self):
```

Comment:
How we avoid this test to be skipped because of ceph health issue as well?

Reply:
good point, I will add an exception to the health check setup fixture